doc/objectives.md

```diff
@@ -0,0 +1,18 @@
+# Study Objectives
+
+- **Is there a relationship between predator size (mass, length, fork-length) and prey size (mass, length, fork-length)?**
+  
+  There will be a trend of larger predators consuming larger prey. 
+  
+- **Does the same species of predator in different geographic locations exhibit the same feeding pattern? Are they eating the same sizes of prey in different locations?**
+  
+  Locations with more similar environmental factors will have predators exhibiting similar feeding preferences. 
+  
```

Comment:
I would maybe focus on predator characteristics over environmental factors in this case, because the next 2 hypotheses focus on the environment more. Something like:
-We'd expect the species of prey available to a certain species of predator to vary across space. This might cause changes in the predator-prey size/taxon relationship as prey availability differs.  For example, Albacore living off different continental shelves around the world do not all have access to the same prey. Some might feed on Pearlside, while others on Atlantic saury. Given this information, we'd predict that Albacore living off continental shelves closer to one another will eat more similar species of a more similar size.